lib/auth/tun.go

```diff
@@ -590,8 +601,19 @@ func (c *TunClient) Close() error {
 
 // GetDialer returns dialer that will connect to auth server API
 func (c *TunClient) GetDialer() AccessPointDialer {
-	return func() (net.Conn, error) {
-		return c.Dial(c.authServers[0].AddrNetwork, "accesspoint:0")
```

Comment:
JFYI, this `AddrNetwork` does not change from server to server, so the loop below does not really affect anything.